aws/resource_aws_launch_template.go

```diff
@@ -567,6 +567,8 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
In the future, per #14601 and the EC2 API documentation we may want to consider flipping this to the new AWS Go SDK `ec2.ResourceType_Values()` function to reduce manual maintenance here. 👍